react/Autosuggest/Autosuggest.js

```diff
@@ -5,9 +5,16 @@ import PropTypes from 'prop-types';
 import classnames from 'classnames';
 import ReactAutosuggest from 'react-autosuggest';
 import IsolatedScroll from 'react-isolated-scroll';
-import omit from 'lodash.omit';
+
+import get from 'lodash/get';
+import invoke from 'lodash/invoke';
+import omit from 'lodash/omit';
 
 import TextField from '../TextField/TextField';
+import smoothScroll from '../SmoothScroll/SmoothScroll';
```

Comment:
Couple of things:
- This is named like a component, even though it's a util. Should start with a lowercase letter.
- This is in the components folder, when it should probably just be in the `Autosuggest` folder for now.